docs/pages/database-access/guides/elastic.mdx

```diff
@@ -0,0 +1,248 @@
+---
+title: Database Access with Elasticsearch
+description: How to configure Teleport Database Access with Elasticsearch.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="10.3"
+>
+  Database access for Elasticsearch is available starting from Teleport `10.3`.
+</Details>
+
+This guide will help you to configure secured access to an Elasticsearch database using Teleport Dabatase Access.
+
+## Prerequisites
+
+- A self-hosted Elasticsearch database. Elastic cloud [does not support client certificates](https://www.elastic.co/guide/en/cloud/current/ec-restrictions.html#ec-restrictions-security).
+
+- A host where you will run the Teleport Database Service. Teleport version 10.3
+  or newer must be installed. <ScopedBlock scope={["oss", "enterprise"]}> This can also be the same instance of Teleport running your Auth and/or Proxy service(s).</ScopedBlock>
+
+  See [Installation](../../installation.mdx) for details.
+
+(!docs/pages/includes/user-client-prereqs.mdx!)
+
+## Step 1/5. Install and configure Teleport
+
+### Set up the Teleport Auth and Proxy Services
+
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+
+### Set up the Teleport Database Service
+
+(!docs/pages/includes/database-access/token.mdx!)
+
+Install Teleport on the host where you will run the Teleport Database Service. Make sure you've selected your installation type (OSS, Enterprise, Clo):
+
+(!docs/pages/includes/install-linux.mdx!)
+
+Start the Teleport Database Service, pointing the `--auth-server` flag to the
+address of your Teleport Proxy Service:
+
+<Tabs>
+<TabItem label="Standalone Binary">
+
+```code
+$ teleport db start \
+  --token=/tmp/token \
+  --auth-server=teleport.example.com:3080 \
+  --name=myelastic \
+  --protocol=redis \
+  --uri={ELASTICSEARCH_HOST}:9200 \
+  --labels=env=dev
+```
+
+</TabItem>
+<TabItem label="Config File">
+Add the following block to `teleport.yaml`
+```yaml
+db_service:
+  enabled: true
+  resources:
+  - labels:
+      "*": "*"
+  databases:
+  - name: myelastic
+    protocol: elasticsearch
+    uri: {ELASTICSEARCH_HOST}:9200
+    static_labels:
+      env: dev
+```
+
+Adjust for your environment, then start or restart Teleport.
+
+</TabItem>
+</Tabs>
+
+<Admonition type="note">
+
+The `--auth-server` flag must point to the Teleport cluster's Proxy Service
+endpoint because the Database Service always connects back to the cluster over a
+reverse tunnel.
+
+</Admonition>
+
+<Admonition type="tip">
```

Comment:
Do we need this tip if we have the the "Config File" tab above?

Reply:
This tip was in the Redis guide that I used as a template, and then I added the Tab above. If we like the Tab I'll remove this, or vice versa.

I'll address this after we resolve @zmb3's two comments above.